docs/code-snippets/office-snippets.yaml

```
@@ -3356,6 +3356,52 @@ Office.Settings#saveAsync:member(2):
     function write(message){
         document.getElementById('message').innerText += message; 
     }
+Office.Tab#id:member:
+  - |-
+    // Office.Tab objects are properties of ribbon updater objects that are passed to the 
+    // Office.ribbon.requestUpdate method. The following shows how to set the visibility of 
+    // a custom contextual tab.
+
+    async function showDataTab() {
+        await Office.ribbon.requestUpdate({
+            tabs: [
+                {
+                    id: "CtxTab1",
+                    visible: true
+                }
+            ]});
+    }
+
+    // The the following does the same thing in TypeScript.
+
+    const showDataTab = async () => {
+        const myContextualTab: Office.Tab = {id: "CtxTab1", visible: true};
```

```suggestion
        const myContextualTab: Office.Tab = { id: "CtxTab1", visible: true };
```
To be consistent with other code.